### PR TITLE
loads and saves preset files

### PR DESCRIPTION
**Pull request by flyingpandaboy**

### CODE REVIEW

File: src/renderer/PresetsBar.tsx [hunk 1]
```diff
@@ -114,28 +116,41 @@ const PresetsBar = () => {
...
   const handleChangeNewPresetName = (newValue: string) => {
     setPresetName(newValue);
 
-    // Update selectedPresetName if the input value matches an existing preset name
-    const newSelectedName = presetNames.some((n) => n === newValue)
-      ? newValue
-      : undefined;
-    setSelectedPresetName(newSelectedName);
+    // TODO: If the preset name is an existing one, we should set the selected one to be that so we can load it
+    setSelectedPresetName(undefined);
```
Comment (selinali2010) [inline src/renderer/PresetsBar.tsx:153]: Oops, another case of my changes being reverted by accident. Could you undo this change?

File: src/renderer/PresetsBar.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import { useMemo, useRef, useState } from 'react';
 import './styles/PresetsBar.scss';
+import { select } from 'd3';
+import { healthCheck, loadPreset, savePreset } from './utils/equalizerApi';
```
Comment (selinali2010) [inline src/renderer/PresetsBar.tsx:4]: nit: remove unused imports

**Review by selinali2010 — CHANGES_REQUESTED** [review]

**Comment by xenown** [comment]
Can you rename the branch to jat-30 and adjust the pr name to match our existing pr formats?
